app/controllers/api/v1/users_controller.rb

```diff
@@ -37,7 +37,7 @@ def request_otp
   def reset_password
     current_user.reset_password(reset_password_digest)
     current_user.save
-    ApprovalNotifierMailer.with(user: current_user).reset_password_approval_email.deliver_later
+    ApprovalNotifierMailer.with(user: current_user).reset_password_approval_email.deliver_now
```

Comment:
I think we should keep this `deliver_later` for now. ActiveJobs are async by default since Rails 5. And we shouldn't block API calls from nurses on sending emails. Agree that we should move towards a persistent queue, with more control in the future.